apps/08jquery-towers-of-hanoi/script.js

```diff
@@ -1,5 +1,38 @@
 'use strict';
-
+var flag = 1;
+var $block = null;
 $(document).ready(function() {
   // Put app logic here
+
+  jQuery('[data-stack]').click(
+
+    function () {
+
+        if(flag === 1){
+          console.log('picking up block');
+          //this is the data stack that was clicked on
+      $block = $(this)
+        .children()
+        .last()
+        .detach();
+      flag = 0;
```

Comment:
Interesting use of a flag variable. It could probably get a better name 😄 but I like it